python/lsst/ts/observatory/control/mock/mtcs_async_mock.py

```diff
@@ -119,6 +120,11 @@ async def setup_types(self) -> None:
             positionCommanded=0.0,
         )
 
+        # MTDome Motion PARKED state. State is set by the mocked park cmd.
+        self._mtdome_evt_azMotion_state = types.SimpleNamespace(
+            state=None, inPosition=False
```

Comment:
I don't think state can ever be `None`. It might be 0, but most likely it is going to be an instance of [MotionState](https://github.com/lsst-ts/ts_xml/blob/a16574f2a257c25557144337a94d9ba50605b017/python/lsst/ts/xml/enums/MTDome.py#L82).

Reply:
I precisely treat None as a non-existing state. The point was that this event state is set a some point during a task.

But I will rethink it again.

Reply:
07a452e0bd700b6b517dbb7c47bdc3e4f90c19be